src/as_execution/execution.rs

```diff
@@ -255,6 +255,7 @@ impl ASContextModule {
                 "assembly_script_local_execution" => Function::new_typed_with_env(store, &fenv, assembly_script_local_execution),
                 "assembly_script_caller_has_write_access" => Function::new_typed_with_env(store, &fenv, assembly_script_caller_has_write_access),
                 "assembly_script_function_exists" => Function::new_typed_with_env(store, &fenv, assembly_script_function_exists),
+                "assembly_script_sha256_hash" =>  Function::new_typed_with_env(store, &fenv, assembly_script_sha256),
```

Comment:
Maybe be coherent and use sha256 everywhere instead of sha256_hash or the opposite (sha256_hash eveywhere)

Reply:
`assembly_script_hash_sha256` will be better and evolutive. 

`assembly_script_hash_xxxxxx`, may be we should rename the existing ABI `assembly_script_hash`